### PR TITLE
feat: block adding liquidity when REACT_APP_BLOCK_POOL_LIQUIDITY is true

### PR DESCRIPTION
**Pull request by Tulun**
Screen Shot 2022-04-08 at 3 35 06 PM

Signed-off-by: Tulun <jaykiraly@gmail.com>

### CODE REVIEW

File: src/components/PoolForm/AddLiquidityForm.tsx [hunk 1]
```diff
@@ -252,11 +252,13 @@ const AddLiquidityForm: FC<Props> = ({
           disabled={
             (!provider || !!formError || Number(amount) <= 0) && isConnected
           }
-          onClick={() =>
-            approveOrPoolTransactionHandler().catch((err) =>
+          onClick={() => {
+            // Block adding liqudiity in app if REACT_APP_BLOCK_POOL_LIQUIDITY is true
+            if (process.env.REACT_APP_BLOCK_POOL_LIQUIDITY) return false;
```
Comment (daywiss) [inline src/components/PoolForm/AddLiquidityForm.tsx:257]: i feel like `process.env.REACT_APP_BLOCK_POOL_LIQUIDITY` should be set to a variable in the constants file and then imported.

Comment (Gamaranto) [reply src/components/PoolForm/AddLiquidityForm.tsx:257]: +1

File: src/components/PoolForm/PoolForm.styles.ts [hunk 2]
```diff
@@ -77,3 +77,26 @@ export const ROIItem = styled.div`
...
+export const LiquidityBlocked = styled.div`
+  backdrop-filter: blur(6px);
+  height: 254px;
+  width: 440px;
+  position: absolute;
+  margin: -20px;
+  z-index: 1000;
+  display: flex;
+  > div {
+    font-family: "Barlow";
```
Comment (Gamaranto) [inline src/components/PoolForm/PoolForm.styles.ts:90]: nit, I think this can be removed

**Review by Gamaranto — COMMENTED** [review]
agree with David comment, otherwise lgtm

File: src/components/PoolForm/AddLiquidityForm.tsx (hunk 1 above)
Comment (Tulun) [reply src/components/PoolForm/AddLiquidityForm.tsx:257]: Ok!

File: src/components/PoolForm/PoolForm.styles.ts (hunk 2 above)
Comment (Tulun) [reply src/components/PoolForm/PoolForm.styles.ts:90]: removed.

**Review by daywiss — APPROVED** [review]

**Comment by Tommy1231232** [comment]
@Tulun can we add a period at the end of "when v2 pools are launched"

**Comment by Tulun** [comment]
> @Tulun can we add a period at the end of "when v2 pools are launched"

Done.

**Review by amateima — APPROVED** [review]

**Pull request merged**